template/node12/Dockerfile

```diff
@@ -60,6 +60,8 @@ ENV exec_timeout="10s"
 ENV write_timeout="15s"
 ENV read_timeout="15s"
 
+EXPOSE 8080
```

Comment:
What was this doing for you? I don't mind merging it if you have a use-case that was blocked by it not being there.